### ISSUE TITLE
Make Knob's default color be the primary one

### ISSUE THREAD

**Issue by beeplin** [quasarframework/quasar#414]
After changing the global primary color, I got this: 
![Uploading image.png…]()

**Comment by rstoenescu** [comment]
Can you upload the image again pls?

**Comment by beeplin** [comment]
Sorry about the corrupt image.

![image](https://cloud.githubusercontent.com/assets/13058150/23305222/ab57d9c6-fad8-11e6-845a-83e87a4be1d4.png)

**Comment by beeplin** [comment]
I changed the primary color in `ap.variables.styl` but the knob color doesn't change. I tried to use `<knob class="text-primary">` in the template or `.knob {color: $primary}` in the style but failed. Now I have to use `knob`'s `color` props to hard-code the color value `#xxxxxx` in the template.

**Comment by rstoenescu** [comment]
You can make use of the `color` Vue prop for Knob. http://quasar-framework.org/components/knob.html#Vue-Properties

But you need to specify it in hexa/rgb/etc, not as "primary" or "secondary" etc so NOT a color from Quasar palette.

**Comment by beeplin** [comment]
yeah that is my solution for the time being. But I dont think that is neat. Most components follow the stylus theme color, and I think Knob should do the same.

**Comment by rstoenescu** [comment]
Will see about this. It's not so easy though and I'm currently overloaded.

**Comment by beeplin** [comment]
no rush~ and REALLY thank you for this wonderful framework~

**Comment by rstoenescu** [comment]
Didn't make it the primary one, but it now by default inherits the CSS "color" prop. Some examples:
```html
<q-knob style="color: red" .... />
<q-knob color="red" ... />
<q-knob class="text-primary" ... />
```

Will be available in v0.14
